daemon/container_operations.go

```diff
@@ -1014,7 +1014,7 @@ func (daemon *Daemon) releaseNetwork(container *container.Container) {
 	if daemon.netController == nil {
 		return
 	}
-	if container.HostConfig.NetworkMode.IsContainer() || container.Config.NetworkDisabled {
+	if container.HostConfig.NetworkMode.IsContainer() {
```

Comment:
I noticed that `container.NetworkSettings` is a pointer, so could (potentially) be `nil`; perhaps we need a check for that below, to prevent a panic if that ever happens (not sure if possible)

Reply:
I'm working on this. And there is another situation where we create a container and connect to network.

Reply:
> I noticed that container.NetworkSettings is a pointer, so could (potentially) be nil; perhaps we need a check for that below, to prevent a panic if that ever happens (not sure if possible)

`container.NetworkSettings` is never `nil`, unless someone update the container config on disk and restart docker daemon. It's always good to check.